src/localization/locale/en.ts

```diff
@@ -175,5 +175,6 @@ export const en = {
     taskInput: 'Task inputs show parameter keys and can also show task run relationships.',
     workPoolInfrastructureConfigurationInstructions: 'Below you can configure workers\' behavior when executing flow runs from this work pool. You can use the editor in the **Advanced** section to modify the existing configuration options if you need additional configuration options.\nIf you don\'t need to change the default behavior, hit **Create** to create your work pool!',
     workPoolInfrastructureConfigurationAgent: 'Prefect Agents handle infrastructure configuration via infrastructure blocks attached to deployments. You can hit **Create** to create this work pool and then head over to the **Blocks** tab to create an infrastructure block for your deployments.\nTo learn more about how to configure infrastructure for Prefect Agents, check out the [docs](https://docs.prefect.io/latest/concepts/infrastructure/).',
+    disableFlowRunCancel: 'Prefect only offers cancellation of runs directly created by a deployment.',
```

Comment:
```suggestion
    disableFlowRunCancel: 'Only runs created from a deployment can be cancelled',
```
This statement has the wrong emphasis, maybe we shift the tone a little?

Reply:
+1 to this rephrasing.